### PR TITLE
TASK: Non-strings are not valid Json.

### PR DESCRIPTION
**Pull request by rolandschuetz**

### CODE REVIEW

File: TYPO3.Neos/Resources/Public/JavaScript/Shared/Utility.js [hunk 1]
```diff
@@ -13,6 +13,9 @@ define(function() {
 		 * @return {boolean}
 		 */
 		isValidJsonString: function(jsonString) {
+			if(typeof jsonString !== 'string') {
```
Comment (aertmann) [inline TYPO3.Neos/Resources/Public/JavaScript/Shared/Utility.js]: Hey, could you use `if (` since that's what's used in the existing code. Thanks

**Comment by aertmann** [comment]
makes sense I guess

File: TYPO3.Neos/Resources/Public/JavaScript/Shared/Utility.js (hunk 1 above)
Comment (rolandschuetz) [inline TYPO3.Neos/Resources/Public/JavaScript/Shared/Utility.js]: Makes sense. Is there an editorconfig or jscs for that?

Comment (aertmann) [inline TYPO3.Neos/Resources/Public/JavaScript/Shared/Utility.js]: nope, haven't really done much of that since all that stuff is happening in the rewrite

**Comment by skurfuerst** [comment]
Hey @rolandschuetz,

this change totally makes sense. Just a quick question - could you provide some background on which error is fixed by the change so I can reproduce it? :-)

Thanks,
Sebastian

**Comment by rolandschuetz** [comment]
This doesn't directly fixes an error, When I was debugging the asset editor and called _readAndDeserializeValue once too often, it lead to an error, cause it tried to check if an object is valid json.

You could say it "fixes" the abort criterial, used https://github.com/neos/neos-development-collection/blob/master/TYPO3.Neos/Resources/Public/JavaScript/Content/Inspector/Editors/AssetEditor.js#L131 and https://github.com/neos/neos-development-collection/blob/master/TYPO3.Neos/Resources/Public/JavaScript/Content/Inspector/Editors/ImageEditor.js#L861

These are the only places in the code where this function is used.

**Pull request merged**